zulipterminal/ui_tools/views.py

```diff
@@ -706,10 +717,12 @@ def update_user_list(
             else:
                 users_display = users
 
-            self.empty_search = len(users_display) == 0
+            self.search_status = (
+                SearchStatus.EMPTY if len(users_display) == 0 else SearchStatus.FILTERED
+            )
 
             # FIXME Update log directly?
```

Comment:
I'm wondering if this `FIXME` is now obsolete. I'm not sure on the idea behind that comment, but going towards an `Enum` seems to not allow updating `log` directly anymore anyways.

Reply:
I'm not certain but imo it should be fine.
This enum merely replaces the boolean that was already being used there, so it shouldn't add any new restrictions?

I assume that the comment (added in [this commit](https://github.com/zulip/zulip-terminal/commit/92aaa8a3d07c8e14d7bffa17cb0d44cc09f940f1)) means moving some of the logic from RightColumnView to UsersView which holds log. And this should be fine, as we already support TopicsView and StreamsView with the new enum, instead of the LeftColumnView. So, it would probably require just moving the enum to a different class.